obspy/io/nordic/core.py

```diff
@@ -1109,7 +1109,8 @@ def nordpick(event):
                 else:
                     amp = None
                 coda = ' '
-                if amplitude.magnitude_hint.upper() == 'ML':
+                mag_hint = (amplitude.magnitude_hint or amplitude.type)
+                if mag_hint is not None and mag_hint.upper() in ['AML', 'ML']:
                     phase_hint = 'IAML'
```

Comment:
Why this is mapping `'ML'` to `'IAML'` in the first place?

Reply:
I've done this because that is how local magnitude picks are read in from Nordic (e.g. I've done this because I did this elsewhere) - When picking amplitudes for local magnitudes in seisan they are stored in the Nordic file as picks of type IAML.
If you are happy with this then I'm good to merge?

Reply:
euhm.. I only realized now.. you're setting this on `phase_hint` even, which is supposed to hold a wave onset pick, not a magnitude type identification.. so is this misusing the `Pick` object to store the timing info of an amplitude measurement?

Reply:
Ah well, maybe I'm just looking at the QuakeML standards a bit differently and it's OK that way. The `phase_hint` might be slightly misused, but hey if you guys actually using this format think it's fine this way, that's good enough for me.